CONTRIBUTING.md

```diff
@@ -2,38 +2,30 @@
 
 ## Setup
 
-In order to contribute to librespot, you will first need to set up a suitable rust build environment, with the necessary dependenices installed. These instructions will walk you through setting up a simple build environment.
+In order to contribute to librespot, you will first need to set up a suitable Rust build environment, with the necessary dependencies installed. These instructions will walk you through setting up a simple build environment.
 
-You will need to have C compiler, rust, and portaudio installed.
+You will need to have C compiler, Rust, and alsa/portaudio libraries installed.
 
 ### Install Rust
 
-The easiest, and recommended way to get rust setu is to use [rustup](https://rustup.rs). You can install rustup with this command:
+The easiest, and recommended way to get Rust is to use [rustup](https://rustup.rs). You can install `rustup` with this command:
 
 ```bash
 curl https://sh.rustup.rs -sSf | sh
 ```
 
-Follow any prompts it gives you to install rust. Once that’s done, rust is ready to use.
- 
-### Install Other Dependencies
-On debian / ubuntu, the following command will install these dependencies :
-
-```bash
-sudo apt-get install build-essential portaudio19-dev
-```
-
-On Fedora systems, the following command will install these dependencies :
+Follow any prompts it gives you to install Rust. Once that’s done, Rust's standard tools should be setup and ready to use.
 
+#### Additional Rust tools - `rustfmt`
+To ensure a consistent codebase, install [`rustfmt`](https://github.com/rust-lang/rustfmt) via `rustup`.
+This is not optional, as Travis CI is set up to check that code is compliant with this repos style guides.
```

Comment:
s/repos/repo's/ or s/this repos/our/.

Technically they are not style 'guides' if they are mandatory. Maybe 'rules' is more accurate.